charts/openforms/templates/configmap-nginx.yaml

```diff
@@ -91,6 +91,11 @@ data:
         include conf.d/proxy;
       }
 
+      location = /admin/forms/form/import/ {
+        client_max_body_size {{ .Values.settings.maxImportSize }};
```

Comment:
```suggestion
        client_max_body_size {{ .Values.settings.maxImportSize | default "20M" }};
```

Reply:
So it wont break deployments when upgrading chart, but using own values.yml and forget to add this value.